lib/entity_store_sequel/postgres_entity_store.rb

```diff
@@ -195,18 +195,23 @@ def get_entities(ids, options={})
     def get_events(criteria)
       return {} if criteria.empty?
 
-      result = {}
+      query = events.where('1=2')
```

Comment:
May be able to build the query without a hack by using a [union](http://www.rubydoc.info/github/jeremyevans/sequel/Sequel%2FDataset%3Aunion). It is weird there isn't something like a `none` to give you an empty set to add to like this essentially does.